tests/integration/test_client.py

```diff
@@ -1,20 +1,28 @@
+"""Integration test for the client_test C++ program."""
+
 import subprocess
 import time
+from collections.abc import Generator
 from pathlib import Path
-from typing import Tuple
 
+import mysql.connector
 import pytest
 
 from .client import (
     g_storage_url,
-    storage,
 )
 from .utils import g_scheduler_port
 
 
 def start_scheduler_workers(
     storage_url: str, scheduler_port: int
-) -> Tuple[subprocess.Popen, subprocess.Popen, subprocess.Popen]:
+) -> tuple[subprocess.Popen, subprocess.Popen, subprocess.Popen]:
+    """
+    Starts the scheduler and two worker processes.
+    :param storage_url: The JDBC URL of the storage.
+    :param scheduler_port: The port for the scheduler to listen on.
+    :return: A tuple of the scheduler process and two worker processes.
```

Comment:
```suggestion
    :return: A tuple of started processes containing:
      - The scheduler process.
      - The first worker process.
      - The second worker process.
```
We usually use bullet points to document pairs and tuples. For example: https://docs.yscope.com/clp-ffi-py/main/api/clp_ffi_py.ir.html#clp_ffi_py.ir.KeyValuePairLogEvent.to_dict